Jenkinsfile_nightly

```diff
@@ -0,0 +1,25 @@
+#!groovy
+
+properties([
+        pipelineTriggers([cron('00 20 * * *')]),
+        parameters([
+                string(name: 'URL_TO_TEST', defaultValue: 'https://benefit-appeal.nonprod.platform.hmcts.net', description: 'The URL you want to run these tests against'),
```

Comment:
This is cool. I am wondering if we could use this same approach to run our smoke and functional tests in AAT against the external url